src/stylesheets/components/_buttons.scss

```diff
@@ -110,3 +110,30 @@
   border-width: 1px;
   border-radius: 0 4px 4px 0;
 }
+
+/*--- Active styles ---*/
+
+.open > .btn--ghost--inverse {
+  @include button($white, $grey-1, $grey-2);
+}
+
+/*--- Toggle styles ---*/
+
+.toggle--menu {
+  > [class*="btn--"] {
+    margin-bottom: 0;
+  }
+  .btn--close {
+    float: right;
+    @include links($grey-1, none, $grey-2, $grey-1);
+    @include milli;
+    margin-bottom: 0;
+  }
+}
+
+body:not(.has-js) .toggle--menu {
```

Comment:
I think we need rid ourselves of the `:not()` pseudo class by having classes for each state (`.has-js` or `.no-js`) that we can target

Reply:
Will skip this for now as it's a larger project and will be better to treat as a separate project.